source/common/protobuf/utility.cc

```diff
@@ -610,10 +610,12 @@ ProtobufWkt::Value ValueUtil::structValue(const ProtobufWkt::Struct& obj) {
 }
 
 ProtobufWkt::Value ValueUtil::listValue(const std::vector<ProtobufWkt::Value>& values) {
-  ProtobufWkt::Value val;
+  auto* list = new ProtobufWkt::ListValue();
```

Comment:
Use std::make_unique here and release it when passing it into set_allocated_list_value. That way if there's an early exit or an exception you won't leak.

Reply:
Done. 3ffbd56. Thanks!